calico-enterprise/getting-started/install-on-clusters/kubernetes/helm.mdx

```diff
@@ -38,7 +33,227 @@ In this guide, you install the Tigera Calico operator and custom resource defini
 
 ### Download the Helm chart
 
-<GettingStartedInstallOnClustersKubernetesHelm />
+1. Add the Calico helm repo:
+
+  ```bash
+  helm repo add projectcalico https://docs.tigera.io/calico/charts
+  ```
+
+### Customize the Helm chart
+
+If you are installing on a cluster installed by EKS, GKE, AKS or Mirantis Kubernetes Engine (MKE), or you need to customize TLS certificates, you **must** customize this Helm chart by creating a `values.yaml` file. Otherwise, you can skip this step.
+
+1. If you are installing on a cluster installed by EKS, GKE, AKS or Mirantis Kubernetes Engine (MKE), set the `kubernetesProvider` as described in the [Installation reference](../../../reference/installation/api.mdx#operator.tigera.io/v1.Provider). For example:
+
+  ```bash
+  echo '{ installation: {kubernetesProvider: EKS }}' > values.yaml
+  ```
+
+   For Azure AKS cluster with no Kubernetes CNI pre-installed, create `values.yaml` with the following command:
+
+  ```yaml
+  cat > values.yaml <<EOF
+  installation:
+    kubernetesProvider: AKS
+    cni:
+      type: Calico
+    calicoNetwork:
+      bgp: Disabled
+      ipPools:
+      - cidr: 10.244.0.0/16
+        encapsulation: VXLAN
+  EOF
+  ```
+
+1. Add any other customizations you require to `values.yaml`. To see values that can be customized in the chart, see the [helm docs](https://helm.sh/docs/) or run the following command:
+
+  ```
+  helm show values projectcalico/tigera-operator --version {{releaseTitle}}
```

Comment:
For master, the version must be v0.0. So when this doc is ready, make sure the master version of this page is slightly different.

Reply:
Thanks, there is now a if-else addressing the `master` version case